components/researcher/data/Line.jsx

```diff
@@ -43,6 +43,13 @@ const Line = ({ data, title, units }) => {
               hideTicks
               orientation="bottom"
               numTicks={4}
+              tickFormat={(date) => {
+                return new Intl.DateTimeFormat("default", {
```

Comment:
this can be cleaned up with something similar 
```jsx
{new Date(date).toLocaleTimeString(navigator.language, {
    hour: "2-digit",
    minute: "2-digit",
})}


```